test/stubs.py

```diff
@@ -98,7 +102,13 @@ def get(self, section, key):
 
     def set(self, section, key, value):
         # print self.sections()
-        self.store['%s.%s' % (section, key)] = value
+        self.store[section][key] = value
+
+    def items(self, section):
+        items_from_store = self.store[section]
+        if len(items_from_store) > 0:
```

Comment:
I'm a little curious about this.  Is there a potential for a `KeyError` here?  Regardless, I think a quick comment about what the intent here is would help since it's not immediately apparent to me at least.